src/main/java/core/basesyntax/service/serviceimpl/FruitReportCreateImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.exception.NegativeBalanceException;
+import core.basesyntax.service.interfaces.FruitReportCreate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitReportCreateImpl implements FruitReportCreate {
+    private static final String SEMICOL = ",";
+
+    public String createReport(HashMap<String, Integer> fruitQuantity) {
```

Comment:
better to add a StorageDao as a field of this class and call it to get the storage info instead of passing it as a param here
```suggestion
    public String createReport() {
```